services/import-service/serverless.ts

```diff
@@ -0,0 +1,56 @@
+import type { AWS } from "@serverless/typescript";
+
+import importProductsFile from "@functions/importProductsFile";
+import importFileParser from "@functions/importFileParser";
+
+const serverlessConfiguration: AWS = {
+  service: "import-service",
+  frameworkVersion: "3",
+  plugins: ["serverless-esbuild", "serverless-offline"],
+  provider: {
+    name: "aws",
+    runtime: "nodejs14.x",
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: "1",
+      NODE_OPTIONS: "--enable-source-maps --stack-trace-limit=1000",
+      IMPORT_BUCKET: "my-shop-import-service",
+    },
+    iam: {
+      role: {
+        statements: [
+          {
+            Effect: "Allow",
+            Action: "s3:ListBucket",
+            Resource: ["arn:aws:s3:::my-shop-import-service"],
+          },
+          {
+            Effect: "Allow",
+            Action: "s3:ListBucket",
```

Comment:
nit: not sure if copy and delete file in s3 will work like this. Sould be Action: "s3:*"